bitrise.yml

```diff
@@ -27,7 +26,7 @@ workflows:
 
   create-release-version:
     envs:
-    - RELEASE_VERSION: 0.1.2
+    - RELEASE_VERSION: 1.0.0
```

Comment:
or actually, just don't change the version number - we'll bump it when we create the new release